pkg/kubernetes/app.js

```diff
@@ -30,6 +30,16 @@ define([
             calculated.containers = "0";
 
             /*
+             * Calculate total number of containers = replicas * containersArray.length
+             */
+            client.select(service.spec.selector || { },
+                          service.metadata.namespace, "ReplicationController").forEach(function(rc) {
```

Comment:
I would suggest sticking with the previous approach where we enumerate the pods. Each pod has a spec which says how many containers are expected in that pod. We would just use that number.